src/main/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorUtil.groovy

```diff
@@ -91,13 +92,31 @@ class IqPolicyEvaluatorUtil
     if (!isNetworkError || failBuildOnNetworkError) {
       throw e
     }
+    else if (isHttp404Error(e)) {
+      listener.logger.println Messages._IqPolicyEvaluation_ResourceNotFound()
+      run.result = Result.FAILURE
+      return null
+    }
     else {
       listener.logger.println ExceptionUtils.getStackTrace(e)
       run.result = Result.UNSTABLE
       return null
     }
   }
 
+  private static boolean isHttp404Error(final Exception throwable) {
+    boolean result = false
+    int httpResponseExceptionIndex = ExceptionUtils.indexOfType(throwable, HttpResponseException)
+    if (httpResponseExceptionIndex >= 0) {
+      Throwable[] throwables = ExceptionUtils.getThrowables(throwable)
+      HttpResponseException httpResponseException = (HttpResponseException) throwables[httpResponseExceptionIndex]
+      if (httpResponseException.statusCode == 404) {
```

Comment:
Minor.  Personal pet peeve.  I think you could just

```java
return (httpResponseException.statusCode == 404)
```
Maybe get rid if the result variable altogether and just return false if we fell through to there?

Reply:
> Maybe get rid if the result variable altogether and just return false if we fell through to there?

Changed e299be4